### DISCUSSION TITLE
Changelist TC 4024.25 to 4024.29

### DISCUSSION THREAD

**Discussion by cfe-arjo** [Roald87/TwinCatChangelog#51] [General]
Got this from Beckhoff support team :

[Changelist_4024.25_to_4024.29.pdf](https://github.com/Roald87/TwinCatChangelog/files/9206317/Changelist_4024.25_to_4024.29.pdf)

**Comment by Roald87** [comment]
Nice thanks! I'll add them next week, because I'm on holidays currently. 

Why did you get the change log? Earlier they never did this.

**Reply by cfe-arjo** [reply]
I asked for a bugfix at every TC release since one year, I gess Beckhoff support team got bored of answering to my email... 😄 

They didn't wanted to send me the changelog for 4024.35.
